profile/README.md

```diff
@@ -1 +1,46 @@
+- [massa-sc-compiler](https://github.com/massalabs/massa-sc-compiler): Compile a smart-contract to a WASM binary.
+- [massa-sc-playground](https://github.com/massalabs/massa-sc-playground): Playground in AssemblyScript for smart-contracts.
+- [massa-web3](https://github.com/massalabs/massa-web3): Library to help the implementation of Web3 applications on top of the Massa blockchain.
+- [massa-as-gaming-sdk](https://github.com/massalabs/massa-as-gaming-sdk): SDK for smart-contracts in AssemblyScript made for game development.
+- [massa-standards](https://github.com/massalabs/massa-standards): Code implementing standard smart-contracts, such as NFT or ERC-20.
+- [massa-sc-examples](https://github.com/massalabs/massa-sc-examples): Smart contract examples.
+- [as](https://github.com/massalabs/as): Utils to write smart-contracts in AssemblyScript.
+
+## Clients
+- [Thyra](https://github.com/massalabs/thyra): User-friendly client connected to the Massa blockchain to perform common operations. Accepts plugins to add more fonctionalities.
+- [thyra-plugin-store](https://github.com/massalabs/thyra-plugin-store): MassaLabs's list official Thyra plugins supported.
+- [thyra-plugin-wallet](https://github.com/massalabs/thyra-plugin-wallet): Plugin for Thyra implementing a wallet.
+- [thyra-plugin-blogNoCode](https://github.com/massalabs/thyra-plugin-blogNoCode): Plugin for Thyra to create a blog without coding, allowing to store the website on the blockchain.
+- [thyra-node-manager-plugin](https://github.com/massalabs/thyra-node-manager-plugin): Plugin for Thyra to manage a remote node from a user-friendly interface.
+- [thyra-playground-plugin](https://github.com/massalabs/thyra-playground-plugin): Plugin for Thyra to serve a playground for smart-contracts development.
+- [thyra-plugin-massa-core](https://github.com/massalabs/thyra-plugin-massa-core): Plugin for Thyra implementing Massa core features.
+- [thyra-plugin-hello-world](https://github.com/massalabs/thyra-plugin-hello-world): HelloWorld example for plugin developers.
+- [thyra-menu-bar-app](https://github.com/massalabs/Thyra-Menu-Bar-App): Menu bar application icon for Thyra.
+
+## Tests and benchmarks
+
+- [massa-benchmark](https://github.com/massalabs/massa-benchmark-main): Contains benchmarks of some parts of the massa blockchain.
+- [massa-unit-tests-src](https://github.com/massalabs/massa-unit-tests-src): Source of every smart-contracts serving as unit tests inside the [massa-core](https://github.com/massalabs/massa) repository.
+- [massa-sc-tester](https://github.com/massalabs/massa-sc-tester): Minimal testing environment for smart-contracts.
+
+## Documentation
+
+- [massa-docs](https://github.com/massalabs/massa-docs): Contains the documentation and code to build [the massa-docs website](https://docs.massa.net/en/latest/).
+- [docu-dev](https://github.com/massalabs/docu-dev): Documentation for the developers.
```

Comment:
I doubt that we should mention 2 documentation repos, as the docu-dev should superseed massa-docs, if I am not mistaken. @gregLibert Pls validate if we should mention both repos in Massa organization Readme.